### PR TITLE
Linux Mac installation environment variables

### PR DESCRIPTION
**Pull request by baturozgur**
Added description for ED_ENV_VARS in linux and mac installation and provided sample.
Added sudos to commands that will always require sudo.

### CODE REVIEW

File: installation/linux.md [hunk 1]
```diff
@@ -6,42 +6,51 @@ description: >-
...
 The installation process deploys Edge Delta into the path`/opt/edgedelta/agent/` and system service `edgedelta` starts automatically with default configuration.
 
+ED_ENV_VARS special variable is used as part of the installation command to pass one or more persistent environment variables to the agent which will run as the system service.
+
+```bash
+sudo ED_API_KEY=<your api key> \
+ED_ENV_VARS="MY_VAR1=MY_VALUE_1,MY_VAR2=MY_VALUE_2" \
+bash -c "$(curl -L https://release.edgedelta.com/release/install.sh)"
+```
+
```
Comment (taylanisikdemir) [inline installation/linux.md:41]: somewhere here you can link the environment-variables page.
"See [environment-variables] page for full list of variables supported by agent"

File: installation/linux.md [hunk 2]
```diff
@@ -6,42 +6,51 @@ description: >-
...
 ## Troubleshooting
 
 Check the agent's service status using one of the following commands \(dependent on Linux distribution\):
 
 ```text
+sudo su
 service edgedelta status
 systemctl status edgedelta
 /etc/init.d/edgedelta status
```
Comment (taylanisikdemir) [inline installation/linux.md]: this returns " No such file or directory" on my linux

File: installation/linux.md [hunk 3]
```diff
@@ -64,7 +73,7 @@ Make sure to run uninstallation process as root.
 systemctl stop edgedelta.service
 systemctl disable edgedelta.service
 rm /etc/systemd/system/edgedelta.service
-rm /usr/lib/systemd/system/edgedelta.service 
+rm /usr/lib/systemd/system/edgedelta.service
```
Comment (taylanisikdemir) [inline installation/linux.md]: I guess some of these commands will need `sudo` as well

**Review by taylanisikdemir — APPROVED** [review]

File: installation/linux.md (hunk 2 above)
Comment (baturozgur) [reply installation/linux.md]: Yeah it lists 3 alternative commands depending on linux distro as described in line 43.

Comment (baturozgur) [reply installation/linux.md]: I removed, sudo su it is confusing there,  added sudo to each command.

File: installation/linux.md (hunk 3 above)
Comment (baturozgur) [reply installation/linux.md]: I removed this section and added uninstall script with sudo.

File: installation/linux.md (hunk 1 above)
Comment (baturozgur) [reply installation/linux.md:41]: Done.

**Pull request merged**